docs/tools/piral-inspector.md

```diff
@@ -4,4 +4,203 @@ title: Piral Inspector
+
+Piral Inspector is now available, and you can open it via the Developer Tools (`Developer > Developer Tools`).
+
+On the tools bar, there now will also appear the Piral icon, and with a click on it, you will see the initial "Not connected" page. 
+
+In case you are currently running a Piral Instance in debug mode, you will see the available Pilets and registered routes.
+
+![OperaExtensionToolPiralInspector](../diagrams/piralinspector_chrome_add-on_tool.png)
+:::
+
+## Using Piral Inspector
+
+Piral Inspector offers a wide variety of capabilities regarding Piral Instance and its corresponding Pilets.
+
+The add-on/extension operates on a locally in debug mode running Piral Instance, and it directly interacts with it.
+
+The following sections are shown inside the extension:
+- Information about Piral Instance
+- Available Pilets
+- Add Pilets
+- Registered Routes
+- Events
+- State Container
+
+All of these sections will documented in the following chapters:
+
+### Piral Instance
+In the very left top, the extension shows the name and the version of the currently running Piral Instance.
+Besides that, in the very right top, a click on the icon allows the user to adapt the extension's settings to its needs.
+
+![PiralInstanceToolPiralInspector](../diagrams/piralinspector_tool_piralinstance.png)
+
+In the debug setting, the user can activate/deactivate the following options:
+
+![DebugSettingsToolPiralInspector](../diagrams/piralinspector_tool_debugsettings.png)
+
+#### State Container Logging
+Activates/Deactivates the logging of changes in the state container to the Developer Tools console.
+
+#### Load available Pilets
+Activates/Deactivates the loading of the available Pilets.
+::: warning: Only for Pilets
+This feature has an impact only when the emulator is used, i.e., when `pilet debug` is running.
+:::
+
+#### Full refresh on change
+Refreshes the current browser page in case there was a change in the debugged Pilet.
+::: warning: Only for Pilets
+This feature has an impact only when the emulator is used, i.e., when `pilet debug` is running.
+:::
+
+#### Visualize component origins
+Activates the option to visualize the origin of the modules as another icon in Piral Instance section:
+
+![DebugSettingsVisCompOriginToolPiralInspector](../diagrams/piralinspector_tool_visualizecomponentoriginicon.png)
+
+After clicking on the icon, the origin of each single tile will be displayed:
+
+![DebugSettingsVisCompOriginToolPiralInspector](../diagrams/piralinspector_tool_visualizecomponentorigintile.png)
+
+The origin will also be displayed when hovering over a tile.
+
+
+### Available Pilets
+The Piral Inspector lists all the currently available Pilets. Like illustrated in the table below.
+
+![AvailablePiletsToolPiralInspector](../diagrams/piralinspector_tool_availablePilets.png)
+
+By clicking on the switch button on the left, the user can activate and deactivate each Pilet.
```

Comment:
Pilet should be written like "pilet" (lowercase), while Piral is written like "Piral" (uppercase).

Reply:
Pilet is name like Piral is.
What is the difference?
Lothar is curious about the difference.

Reply:
We discussed that a long time ago. Actually, even without an explanation "consistency" should be key (look at *all* the documentation).

Piral is a proper name (like "Bono") while pilet is a concept / normal name (like "microservice").